timemachine/fe/rbfe.py

```diff
@@ -400,6 +444,126 @@ def estimate_relative_free_energy(
         raise err
 
 
+def estimate_relative_free_energy_via_greedy_bisection(
+    mol_a,
+    mol_b,
+    core: NDArray,
+    ff: Forcefield,
+    host_config: Optional[HostConfig],
+    seed: int,
+    n_frames: int = 1000,
+    prefix: str = "",
+    lambda_interval: Optional[Tuple[float, float]] = None,
```

Comment:
I considered this and was on the fence; I went with optional for consistency with `n_windows`, and because this method composes a bit more nicely with calling functions that also take an optional `lambda_interval` (e.g. if we chose a default here, I think `run_complex` would have to either not support overriding this, or hardcode its own default). That said, we don't currently allow overriding `lambda_interval`, and maybe there's no good reason to support this?

Reply:
Ahh, sorry myopic comment -- didn't consider other functions where this is optional. Good as-is!